packages/devtools_app/pubspec.yaml

```diff
@@ -11,17 +11,16 @@ version: 0.8.1-dev.1
 homepage: https://github.com/flutter/devtools
 
 environment:
-  sdk: '>=2.6.0 <3.0.0'
+  sdk: ">=2.6.0 <3.0.0"
   # The flutter desktop support interacts with build scripts on the Flutter
   # side that are not yet stable, so it requires a very recent version of
   # Flutter. This version will increase regularly as the build scripts change.
-  flutter: '>=1.10.0'
+  flutter: ">=1.10.0"
```

Comment:
ditto - can you undo any formatting changes that are unrelated to bumping the vm_snapshot_analysis version number?

Reply:
Done